tests/ReproduceTest.php

```diff
@@ -0,0 +1,195 @@
+<?php
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+session_start();
+header('Content-type: text/plain');
+
+require_once(__DIR__ . '/vendor/autoload.php');
+require_once(__DIR__ . '/../vendor/autoload.php');
+
+use Google\Cloud\Spanner\V1\SpannerGrpcClient;
+use Google\Cloud\Spanner\V1\CreateSessionRequest;
+use Google\Cloud\Spanner\V1\DeleteSessionRequest;
+use Google\Auth\ApplicationDefaultCredentials;
+use Google\Cloud\Spanner\V1\ExecuteSqlRequest;
+use Google\Cloud\Spanner\V1\ListSessionsRequest;
+use Google\Cloud\Spanner\V1\ListSessionsResponse;
+
+class ReproduceTest extends PHPUnit_Framework_TestCase
+{
+    public function setUp()
+    {
+        $this->base_rpc = 100;
+        $this->addition_rpc = 50;
+        $this->_DEFAULT_MAX_CHANNELS_PER_TARGET = 10;
+        $this->_WATER_MARK = 100;
+    }
+
+
+    public function tearDown()
+    {
+
+    }
+
+
+    public function createStub($enable_gcp = true)
+    {
+        putenv("GOOGLE_APPLICATION_CREDENTIALS=./grpc-gcp.json");
```

Comment:
Strange that this is here. Would be better to put it in `setUp`, or better yet make it user defined and throw an error if it doesn't exist.

Reply:
Done.